base/docs/Docs.jl

```diff
@@ -675,4 +675,19 @@ function hasdoc(binding::Docs.Binding, sig::Type = Union{})
 end
 
 
+"""
+    undocumented_names(mod::Module; all=false)
```

Comment:
needs to document `identifiers`.  Also should have a `::Bool` type qualifier on the keywords

Reply:
I can't think of any use cases for `identifiers=false` so I just dropped it.